app/controllers/wyeworkers_controller.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+class WyeworkersController < ApplicationController
+  include RestJsonUtils
+
+  before_action :set_wyeworker, only: %i[show update destroy]
+  before_action :explain_no_links_editing, only: %i[create update]
+
+  EXPOSED_PLAIN_ATTRIBUTES = %w[name id].freeze
```

Comment:
Why only this attributes? Why control this in the controller instead of the model?